src/poliastro/core/iod.py

```diff
@@ -61,6 +61,13 @@ def vallado(k, r0, r, tof, short, numiter, rtol):
         True for short path, False for long one.
     tof : ~float
         Time of flight
+    M : int
+        Number of revolutions
+    is_prograde: boolean
+        Controls the desired inclination of the transfer orbit.
+    is_lowpath: boolean
+        Decides between high and low path when two possible transfer orbits are
```

Comment:
What do "high" and "low" mean in this context? (I _think_ I know, but it would be nice to clarify a bit, if it can be done in a short sentence)

Reply:
What about this?

```python
"""
is_lowpath:
    If `True` or `False`, gets the transfer orbit whose vacant focus is below or above the chord line, respectively.
"""
```

Reply:
Looks OK!